spec.md

```diff
@@ -43,6 +43,11 @@ secPrefix:
 lastDelim: ", and"
 ---
 
+# Versions
+
+- 1.0.0 Clarify the versioning scheme of this specification.
+- 0.4.0 Doc moved to Markdown
```

Comment:
This was actually 0.3.0. Could you summarize the release notes here: https://github.com/chipsalliance/firrtl-spec/releases/tag/v0.4.0